source/pdk/1.0/pdk_reference.md

```diff
@@ -69,6 +69,28 @@ Argument   | Description   | Values      | Default
 `--template-url=<GIT_URL>` | Overrides the template to use when generating this defined\_type. | A valid Git URL or path to a local template.   | Uses the template used to generate the module. If that template is not available, the default template at [puppetlabs/pdk-module-template](https://github.com/puppetlabs/pdk-module-template) is used.
 `<defined_type_name>` | **Required**. The name of the defined type to generate. | A defined type name beginning with a lowercase letter and including only lowercase letters, digits, and underscores.    | No default.
 
+## `pdk new task` command
+
+Generates a new task and task metadata in the current module. 
+
+Usage:
+
+```
+pdk new task [--template-url=<GIT_URL>] <task_name>
+```
+
+For example:
+
+```
+cd my_module
+pdk new task my_task
+```
+
+Argument   | Description   | Values      | Default
+----------------|:---------------:|:------------------:|-------------------------
+`--template-url=<GIT_URL>` | Overrides the template to use when generating this task. | A valid Git URL or path to a local template.   | Uses the template used to generate the module. If that template is not available, the default template at [puppetlabs/pdk-module-template](https://github.com/puppetlabs/pdk-module-template) is used.
```

Comment:
"not available" should say "not specified". If the template is specified, but invalid, results are ... undefined. likely a error, or a non-working result. This needs to be fixed in the other mentions of this param too.